lib/amazon-pricing/definitions/instance-type.rb

```diff
@@ -181,6 +181,8 @@ def self.get_values(json, category_type, override_price = false)
     end
 
     @@Name_Lookup = { # basic name and description lookup
+      'a1.medium' =>  'General purpose A1 Medium', 'a1.large' =>  'General purpose A1 Large', 'a1.xlarge' =>  'General purpose A1 Extra Large',
+      'a1.2xlarge' => 'General purpose A1 Double Extra Large', 'a1.4xlarge' => 'General purpose A1 Quadruple Extra Large',
```

Comment:
curious: where is "General purpose" classification coming from?

Reply:
nit: none of our other "General Purpose" types have this in the name

Reply:
Fair nit; think we need to discuss naming convention, as other names we're using include the instance  family in them. We can discuss at the pricing sync.